README.rst

```diff
@@ -1,10 +1,8 @@
 ###############
-pyPodcastParser
+pypodcastparser
 ###############
 
-|pypi| |pip_monthly| |testing| |coverall| |codacy| |license|
-
-pyPodcastParser is a podcast parser. It should parse any RSS file, but it specializes in parsing podcast rss feeds. pyPodcastParser is agnostic about the method you use to get a podcast RSS feed. Most user will be comfortable with the Requests_ library.
+pypodcastparser is a podcast parser. It should parse any RSS file, but it specializes in parsing podcast rss feeds. pypodcastparser is agnostic about the method you use to get a podcast RSS feed. Most user will be comfortable with the Requests_ library.
```

Comment:
Type in the last sentence ("Most user") and it looks like there's an extra space after Requests.

Reply:
Fixed